src/store/feedback/index.js

```diff
@@ -15,6 +21,21 @@ export default {
   actions: {
     async sendFeedback({commit}, feedback) {
       commit('SET_FEEDBACK', feedback)
+      const data = {
+        "lab_id": feedback.lab_id,
+        "service_id": feedback.service_id,
+        "order_id": feedback.orderId,
+        "rating_by": feedback.customer_id,
+        "rating": feedback.rate,
+        "created": feedback.createdAt
+    }
+      axios.post(`${baseUrl}/rating`, data)
```

Comment:
You are already inside an `async` function. I think it would be better to use `await` here. I would usually let this slide but knowing that this is code on the Vuex store I prefer it to be tidy. @luthfiazhari 

And please do not use `console.log` if it is not necessary.